sdk-setup/src/mb2

```diff
@@ -1385,8 +1444,75 @@ run_prepare() {
             -bp --short-circuit \
             "$TMP_SPEC"
     )
-    local rc=$?
-    return $rc
+}
+
+cmake_post_cleanup() {
+    # Handle path changes for Qt Creator
+    local cmakefiles="CMakeCache.txt"
+
+    cp -p CMakeCache.txt CMakeCache.txt.qtccache
+
+    for cbpfile in *.cbp; do
+        [[ -f $cbpfile ]] || continue
+        cp -p "$cbpfile" "$cbpfile.qtccache"
+        cmakefiles="$cmakefiles $cbpfile"
+    done
+
+    if [[ $STATEDIR == /home/mersdk/share* && $MER_SSH_HOST_SHARED_HOME ]]; then
+        sed -i "s|/home/mersdk/share|$MER_SSH_HOST_SHARED_HOME|g" $cmakefiles
+    elif [[ $STATEDIR == /home/src1* && $MER_SSH_HOST_SHARED_SRC ]]; then
+        sed -i "s|/home/src1|$MER_SSH_HOST_SHARED_SRC|g" $cmakefiles
+    else
+        echo "Failed to set CMake paths for Qt Creator"
+        rm *.qtccache
+    fi
+}
+
+cmake_pre_setup() {
+    # Undo path changes for Qt Creator
+    if [[ -f "CMakeCache.txt.qtccache" ]]; then
+        mv CMakeCache.txt.qtccache CMakeCache.txt
+        for cbp_file in *.cbp.qtccache; do
+            mv $cbp_file ${cbp_file%.qtccache}
+        done
+    fi
+}
+
+run_cmake() {
+    local rpmbuild="--build . --target all"
+    local sb2args=( -t "$OPT_TARGET" env PATH="${WRAPPERSDIR}:${USERWRAPPERSDIR}:${PATH}" )
```

Comment:
I would use those literally as it is done under run_qmake

Reply:
fixed